apps/challenges/permissions.py

```diff
@@ -13,8 +13,12 @@ def has_permission(self, request, view):
 
         if request.method in permissions.SAFE_METHODS:
             return True
-        elif request.method in ['DELETE', 'PATCH', 'PUT']:
-            challenge = Challenge.objects.get(pk=request.parser_context['kwargs']['pk'])
+        elif request.method in ['DELETE', 'PATCH', 'PUT', 'POST']:
```

Comment:
If the POST method isn't allowed the first test fails where it is a legitimate disable request via a POST method :confused: